lesson_13/homework_13.py

```diff
@@ -16,3 +16,120 @@
     декількох дій, вивести в кого найбільше енергії лишилося.
 Створити тести на клас та на атрибути класу.
 """
+
+class Human():
+    def __init__(self, name, surname, birthdate, gender):
+        self.name = name
+        self.surname = surname
+        self.birthdate = birthdate
+        self.gender = gender
+        self.energy = 100
+
+    def eating(self):
+        """function ads +5 energy points"""
+        self.energy += 5
+
+    def sleeping(self):
+        """function ads +10 energy points"""
+        self.energy += 10
+
+    def speaking(self):
+        """function takes 5 energy points"""
+        self.energy -= 5
+
+    def walking(self):
+        """function takes 10 energy points"""
+        self.energy -= 10
+
+    def homework(self):
+        """function takes 90 energy points"""
+        self.energy -= 90
+
+
+Alex = Human("Alex", "Petrenko", "12.03.89", 'male')
+Pawlo = Human("Pawlo", "Semenow", "02.11.82", 'male')
+Serhii = Human("Serhii", "Rykov", "22.09.98", 'male')
+Olena = Human("Olena", "Petrenko", "10.03.99", 'female')
+Katya = Human("Olena", "Verzilowa", "22.06.79", 'female')
+
+if __name__ == "__main__":
+    Alex.sleeping(), Alex.speaking()
+    Pawlo.walking(), Pawlo.eating()
+    Serhii.homework()
+    Olena.eating(), Olena.sleeping()
+    Katya.homework(), Katya.sleeping()
+    energy_list = [Alex.energy, Pawlo.energy, Serhii.energy, Olena.energy, Katya.energy]
+    print("Max energy after all actions :", max(energy_list))
```

Comment:
```suggestion
Alex = Human("Alex", "Petrenko", "12.03.89", 'male')
Pawlo = Human("Pawlo", "Semenow", "02.11.82", 'male')
Serhii = Human("Serhii", "Rykov", "22.09.98", 'male')
Olena = Human("Olena", "Petrenko", "10.03.99", 'female')
Katya = Human("Olena", "Verzilowa", "22.06.79", 'female')

if __name__ == "__main__":
    Alex.sleeping(), Alex.speaking()
    Pawlo.walking(), Pawlo.eating()
    Serhii.homework()
    Olena.eating(), Olena.sleeping()
    Katya.homework(), Katya.sleeping()
    energy_list = [Alex.energy, Pawlo.energy, Serhii.energy, Olena.energy, Katya.energy]
    print("Max energy after all actions :", max(energy_list))
```
от так робити не варто, бо у вас тут же тести і виконання тесті в залежит від коду модуля. Краще 
1. винести тести в окремий файл
2. налаштування для тестів робити в `def setUp():`